Docs/source/acknowledge_us.rst

```diff
@@ -5,46 +5,200 @@ Acknowledge WarpX
+
+      .. code-block:: bibtex
+
+          @misc{WarpX,
+            author = {Vay, Jean-Luc and
+                      Acciarri, Marco and
+                      Almgren, Ann and
+                      Amorim, Lígia Diana and
+                      Andriyash, Igor and
+                      Angus, Justin Ray and
+                      Antoun, Thierry and
+                      Belkin, Daniel and
+                      Bizzozero, David and
+                      Blelly, Aurore and
+                      Clark, Stephen Eric and
+                      Dammak, Eya and
+                      Fedeli, Luca and
+                      Formenti, Arianna and
+                      Garten, Marco and
+                      Ge, Lixin and
+                      Giacomel, Lorenzo and
+                      Gott, Kevin and
+                      Giacomel, Lorenzo and
+                      Groenewald, Roelof E. and
+                      Grote, David and
+                      Gu, Junmin and
+                      Harrison, Cyrus and
+                      Haseeb, Muhammad and
+                      Huebl, Axel and
+                      Jambunathan, Revathi and
+                      Kara-Mostefa, Ilian and
+                      Klion, Hannah and
+                      Kumar, Prabhat and
+                      Lehe, Remi and
+                      Loring, Burlen and
+                      Marks, Thomas and
+                      Miller, Phil and
+                      Myers, Andrew and
+                      Pech, Juliette and
+                      Rheaume, Elisa and
+                      Richardson, Glenn and
+                      Rheaume, Elisa and
+                      Rowan, Michael E. and
+                      Sandberg, Ryan Thor and
+                      Scherpelz, Peter and
+                      Sinn, Alexander and
+                      Shapoval, Olga and
+                      Terzani, Davide and
+                      Th{\'e}venet, Maxence and
+                      Weichman, Kale and
+                      Yang, Eloise and
+                      Zaim, Ne{\"i}l and
+                      Zhang, Weiqun and
+                      Zhao, Yinjian and
+                      Zhu, Kevin Z. and
+                      Zoni, Edoardo},
+              title     = {{WarpX: An advanced Particle-In-Cell code}},
+              year      = 2018,
+              publisher = {Zenodo},
+              doi       = {10.5281/zenodo.4571577},
+              url       = {https://doi.org/10.5281/zenodo.4571577},
+              note      = {\url{https://doi.org/10.5281/zenodo.4571577}},
+              howpublished = {https://blast-warpx.github.io}
+           }
+
+Since the WarpX is an actively evolving project, a specific version might be used in your work to ensure reproducibility. You can select a *version-specific DOI* from the `release page <https://github.com/BLAST-WarpX/warpx/releases>`__ and add the version number to the cited title, e.g. for version ``25.10``:
```

Comment:
```suggestion
Since WarpX is an actively evolving project, a specific version might be used in your work to ensure reproducibility. You can select a *version-specific DOI* from the `release page <https://github.com/BLAST-WarpX/warpx/releases>`__ and add the version number to the cited title, e.g. for version ``25.10``:
```